### PR TITLE
add infinite sigma cases to state estimation example

### PR DESCRIPTION
**Pull request by mgovers**

### CODE REVIEW

File: examples/State Estimation Example.ipynb [hunk 1]
```diff
@@ -513,6 +513,7 @@
     "## Observability\n",
     "\n",
     "In order to perform a state estimation the number of measurements should be larger than or equal to the number of unknowns. For each node there are two unknowns: `u` and `u_angle`.\n",
+    "This implies that missing data can be representing by providing infinity as a standard deviation.\n",
```
Comment (petersalemink95) [inline examples/State Estimation Example.ipynb:516]: I don't think the sentence above implies this

File: examples/State Estimation Example.ipynb [hunk 2]
```diff
@@ -523,7 +524,178 @@
     "And\n",
     "\n",
     "$$n_{\\text{measurements}} = n_{\\text{nodes\\_with\\_voltage\\_sensor\\_without\\_angle}} + 2 n_{\\text{nodes\\_with\\_voltage\\_sensor\\_with\\_angle}} + 2 n_{\\text{branches\\_with\\_power\\_sensor}} + 2 n_{\\text{nodes\\_without\\_any\\_appliances\\_connected}} + 2 n_{\\text{nodes\\_with\\_all\\_connected\\_appliances\\_measured\\_by\\_power\\_sensor}}$$\n",
-    "\n"
+    "\n",
+    "In this formula, only measurements with non-negligible contributions should be considered. That means that measurements with infinite variances have vanishing contribution and are to be omitted, regardless of whether those divergences are due to infinite sigmas or unreasonably large ones.\n"
```
Comment (petersalemink95) [inline examples/State Estimation Example.ipynb:528]: I think the "This implies..." sentence was supposed to be here?

File: examples/State Estimation Example.ipynb [hunk 3]
```diff
@@ -549,21 +721,32 @@
...
     "# for each mutation, only one object is specified\n",
-    "sym_voltage_sensor_update[\"id\"] = [[11, 12, 13]] * 3\n",
+    "sym_voltage_sensor_update[\"id\"] = [[11, 12, 13]] * 4\n",
     "sym_voltage_sensor_update[\"u_measured\"] = [\n",
     "    [10402.31, 10023.68, 10095.40],\n",
     "    [10583.20, 9869.85, 10105.28],\n",
+    "    [10482.87, 9850.85, 10140.77],\n",
     "    [10482.87, 9850.85, 10140.77]\n",
     "]\n",
+    "sym_voltage_sensor_update[\"u_sigma\"][2, 2] = np.inf  # disable the third sensor of the third measurement\n",
```
Comment (petersalemink95) [inline examples/State Estimation Example.ipynb:738]: I think it should say "disable the third measurement of the third sensor

Comment (petersalemink95) [reply examples/State Estimation Example.ipynb:738]: Sorry, it's about batch. It should be "disable the third sensor of the third scenario"

File: examples/State Estimation Example.ipynb [hunk 4]
```diff
@@ -549,21 +721,32 @@
...
+    "\n",
+    "sym_power_sensor_update = initialize_array(\"update\", \"sym_power_sensor\", (4, 1))\n",
+    "sym_power_sensor_update[\"id\"] = [21]\n",
+    "sym_power_sensor_update[\"power_sigma\"] = [\n",
+    "    [1.0e3],\n",
+    "    [1.0e3],\n",
+    "    [1.0e3],\n",
+    "    [np.inf],  # disables this sensor on the last measurment\n",
```
Comment (petersalemink95) [inline examples/State Estimation Example.ipynb:746]: Use scenario here as wel, not measurement

File: examples/State Estimation Example.ipynb (hunk 3 above)
Comment (mgovers) [reply examples/State Estimation Example.ipynb:738]: 👍

File: examples/State Estimation Example.ipynb (hunk 2 above)
Comment (mgovers) [reply examples/State Estimation Example.ipynb]: you're right. fixed

**Review by petersalemink95 — APPROVED** [review]

**Pull request merged**